### PR TITLE
Gh 2588/what is safe screen

### PR DESCRIPTION
**Pull request by biafra23**
Handles #2588

Changes proposed in this pull request:
image

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
To remove the border from navigation bar:

```
ViewControllerFactory.removeNavigationBarBorder()
```

File: Multisig/UI/App/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -120,7 +120,7 @@ class OnboardingViewController: UIViewController {
 
     @IBAction func pageChanged(_ sender: Any) {
         let pc = sender as! UIPageControl
-
+        print ("Page changed")
```
Comment (DmitryBespalov) [inline Multisig/UI/App/Onboarding/OnboardingViewController.swift]: debug print

File: Multisig/UI/App/Onboarding/OnboardingViewController.swift [hunk 2]
```diff
@@ -162,22 +162,23 @@ class OnboardingViewController: UIViewController {
 }
 
 extension OnboardingViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+    //UICollectionViewDataSource
```
Comment (DmitryBespalov) [inline Multisig/UI/App/Onboarding/OnboardingViewController.swift]: can be removed

File: Multisig/UI/ClaimToken/Tutorial/WhatIsSafeViewController.swift [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    @IBOutlet weak var totalValueProtectedStackView: UIStackView!
+    @IBOutlet weak var totalSafesCreatedStackView: UIStackView!
+
+    private var stepLabel: UILabel!
+
+    private var onNext: (() -> ())?
+    private var stepNumber: Int = 1
+    private var maxSteps: Int = 4
+
+    var factory: ClaimSafeTokenFlowFactory!
```
Comment (DmitryBespalov) [inline Multisig/UI/ClaimToken/Tutorial/WhatIsSafeViewController.swift]: can be removed

File: Multisig/UI/ClaimToken/Tutorial/WhatIsSafeViewController.swift [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        stepLabel.text = "\(stepNumber) of \(maxSteps)"
+
+        totalSafesCreatedLabel.setStyle(.callout)
+        totalValueProtected.setStyle(.callout)
+        totalValueProtectedStackView.layer.cornerRadius = 10
+        totalSafesCreatedStackView.layer.cornerRadius = 10
+    }
+
+    @IBAction func nextClicked(_ sender: Any) {
+        print("Next button clicked")
```
Comment (DmitryBespalov) [inline Multisig/UI/ClaimToken/Tutorial/WhatIsSafeViewController.swift]: completion block not called

**Comment by DmitryBespalov** [comment]
@DmitryBespalov check the scroll view setup

**Comment by biafra23** [comment]
> To remove the border from navigation bar:
> ViewControllerFactory.removeNavigationBarBorder()

I could not find this method in main branch, so I implemented my own. See: https://github.com/safe-global/safe-ios/pull/2611/files#diff-d7820cd68cc0e0514e244d1e60f177dc0528083cdc4be5901aab445556385da7R88

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**